forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/account/v1_1/standingorders/StandingOrdersApiController.java

```diff
@@ -59,8 +58,6 @@
     private int PAGE_LIMIT_STANDING_ORDERS;
     @Autowired
     private FRStandingOrderRepository frStandingOrderRepository;
-    @Autowired
-    private AccountDataInternalIdFilter accountDataInternalIdFilter;
```

Comment:
Changing the `@Autowired` fields to Constructor based injection would be a real pain as these Controllers are extended multiple times in the Accounts API (e.g. in some places v2.0 is extended right up to v3.1.6), which means you'd have to provide the constructor arguments from each child controller!